### PR TITLE
Add a bunch of not directly tech related words

### PR DESCRIPTION
**Pull request by RachaelBooth**
Possible solves #1, although I'd still want more interesting words.

### CODE REVIEW

File: list.txt [hunk 1]
```diff
@@ -22,3 +22,144 @@ zoom,
...
+Monty Python,
+Fawlty Towers,
+Robin Hood,
+King Arthur,
+Merlin,
+Guy Fawkes,
+Garden of Eden,
+10 Downing Street,
+Buckingham Palace,
+Holyrood,
```
Comment (AndrewYHuang) [inline list.txt:53]: Hollywood

**Review by AndrewYHuang — CHANGES_REQUESTED** [review]
Nice! Just spotted a typo

File: list.txt (hunk 1 above)
Comment (RachaelBooth) [reply list.txt:53]: Actually, this was going for the nickname of the Scottish parliament (which I've double checked has a single l).

Comment (AndrewYHuang) [reply list.txt:53]: Oh! I didn't know that!

**Review by AndrewYHuang — APPROVED** [review]

**Pull request merged**